options/locale/locale_en-US.ini

```diff
@@ -2300,6 +2300,7 @@ settings.dismiss_stale_approvals_desc = When new commits that change the content
 settings.require_signed_commits = Require Signed Commits
 settings.require_signed_commits_desc = Reject pushes to this branch if they are unsigned or unverifiable.
 settings.protect_branch_name_pattern = Protected Branch Name Pattern
+settings.protect_branch_name_pattern_desc = "Protected branch name patterns. See github.com/gobwas/glob documentation for pattern syntax. Examples: main, release/**"
```

Comment:
I think you mean
```suggestion
settings.protect_branch_name_pattern_desc = "Protected branch name patterns. See <a href="github.com/gobwas/glob">the documentation</a> for pattern syntax. Examples: main, release/**"
```

Reply:
And preferably, extract the link location into a variable.

Reply:
I copied them from other translation. It seems all of them doesn't use a tag but the link is there in the UI.

Reply:
Sounds like a bug to me.
And nothing we should promote doing.
We've often enough seen the problems of hardcoded links in the translation already…

I mean, I can somewhat guess that the browsers see the URL and say, ah, they want to link somewhere!
But still, it is technically buggy behavior as you cannot guarantee that every browser treats it as a link.